docs/docs/versions.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Versions
+layout: docs
+---
+
+Because the TSC library is a client for the Tableau Server REST API, you need to confirm that the version of the TSC
+library that you use is compatible with the version of the REST API used by your installation of Tableau Server.
+
+* TOC
+{:toc}
+
+## Display the REST API version
+
+To display the default version of the REST API used by TSC, run the following code:
+
+```py
+import tableauserverclient as TSC
+
+server = TSC.Server('http://SERVER_URL')
+
+print(server.version)
```

Comment:
Not sure how you installed the library matters, this is about the Server API version not the library's version. 

I just filed #99 that *would* make this example useful.